### PR TITLE
Configuration for NetFx/MVC and the NET5 app for Snapshot Debugger

### PR DESCRIPTION
**Pull request by coderandhiker**
Since we're using a programmatic approach to setting up Application Insights, we have to wire up the configuration settings.

### CODE REVIEW

File: src/WWT.Web/Startup.cs [hunk 1]
```diff
@@ -101,6 +102,9 @@ public void ConfigureServices(IServiceCollection services)
                 builder.AddFilter("Swick.Cache", LogLevel.Trace);
                 builder.AddDebug();
             });
+
+            services.AddSnapshotCollector((snapshotConfiguration) => 
+                _configuration.Bind(nameof(SnapshotCollectorConfiguration), snapshotConfiguration));
```
Comment (twsouthwick) [inline src/WWT.Web/Startup.cs:107]: I am against the `nameof(...)` use with config block names as the name of the variable isn't really tied to the config section name.

File: src/WWTMVC5/Global.asax.cs [hunk 2]
```diff
@@ -158,6 +159,7 @@ private static IServiceProvider BuildServiceProvider()
                 }
             });
 
+
```
Comment (twsouthwick) [inline src/WWTMVC5/Global.asax.cs:162]: nit: probably not needed

**Review by twsouthwick — APPROVED** [review]
A couple small things. Otherwise, LGTM

File: src/WWT.Web/Startup.cs (hunk 1 above)
Comment (coderandhiker) [reply src/WWT.Web/Startup.cs:107]: I would encourage you to share that feedback with our official docs on this configuration: https://docs.microsoft.com/en-us/azure/azure-monitor/app/snapshot-debugger-vm, the nameof approach is what we use to inform customers on how to set this up.

Comment (twsouthwick) [reply src/WWT.Web/Startup.cs:107]: Ah that's a shame. I'll provide some feedback there.

**Pull request merged**